site/src/pages/TemplatesPage/TemplatesPage.tsx

```diff
@@ -6,23 +6,24 @@ import { XServiceContext } from "../../xServices/StateContext"
 import { templatesMachine } from "../../xServices/templates/templatesXService"
 import { TemplatesPageView } from "./TemplatesPageView"
 
-const TemplatesPage: React.FC = () => {
+export const TemplatesPage: React.FC = () => {
```

Comment:
I think we need to update the import of `TemplatesPage` in `AppRouter.tsx`.

Reply:
Gah, thanks!